client/gui/translations/ws_desktop_de.ts

```diff
@@ -92,23 +92,23 @@
     <name>CustomMenuWidget</name>
     <message>
         <source>Undo</source>
-        <translation>Aufmachen</translation>
+        <translation>rückgängig machen</translation>
```

Comment:
"Aufmachen" = Open something in germany like a "Tür Aufmachen" ( Open a Door ) 
Rückgängig machen = Undo